python/lsst/pipe/tasks/dcrAssembleCoadd.py

```diff
@@ -0,0 +1,781 @@
+from __future__ import absolute_import, division, print_function
+
+#
+# LSST Data Management System
+# Copyright 2008-2016 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.    See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+#
+
+from collections import namedtuple
+import numpy
+import scipy.ndimage.interpolation
+from lsst.afw.coord.refraction import differentialRefraction
+import lsst.afw.geom as afwGeom
+import lsst.afw.image as afwImage
+import lsst.afw.math as afwMath
+import lsst.coadd.utils as coaddUtils
+import lsst.pex.config as pexConfig
+import lsst.pipe.base as pipeBase
+from lsst.pipe.tasks.assembleCoadd import _subBBoxIter
+from lsst.pipe.tasks.assembleCoadd import AssembleCoaddTask
+from lsst.pipe.tasks.assembleCoadd import CompareWarpAssembleCoaddTask
+from lsst.pipe.tasks.assembleCoadd import CompareWarpAssembleCoaddConfig
+
+__all__ = ["DcrAssembleCoaddTask"]
+
+
+class DcrAssembleCoaddConfig(CompareWarpAssembleCoaddConfig):
+    filterName = pexConfig.Field(
+        dtype=str,
+        doc="Name of the band-defining filter of the observations.",
+        default='g',
+    )
+    bufferSize = pexConfig.Field(
+        dtype=int,
+        doc="Number of pixels to grow the subregion bounding box by.",
+        default=5,
+    )
+    useFFT = pexConfig.Field(
+        dtype=bool,
+        doc="Option to use use Fourier transforms for the convolutions.",
```

Comment:
`use use`

I think this is better written as "Use Fourier transforms for the convolution?".